src/pkg/services/m365/api/events.go

```diff
@@ -289,63 +295,178 @@ func (c Events) GetItem(
 		}
 	)
 
-	event, err = c.Stable.
+	rawURL := fmt.Sprintf(eventExceptionsBetaURLTemplate, userID, itemID)
+	builder := users.NewItemEventsEventItemRequestBuilder(rawURL, c.Stable.Adapter())
+
+	event, err = builder.Get(ctx, config)
+	if err != nil {
+		return nil, nil, graph.Stack(ctx, err)
+	}
+
+	attachments, err := c.getAttachments(ctx, event, immutableIDs, userID, itemID)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	event.SetAttachments(attachments)
+
+	// Fetch attachments for exceptions
+	exceptionOccurrences := event.GetAdditionalData()["exceptionOccurrences"]
+	if exceptionOccurrences != nil {
```

Comment:
Another chance to reduce nesting.
```suggestion
	if exceptionOccurrences == nil {
	  return event, EventInfo(event), nil
	}
```